api/envoy/extensions/filters/listener/http_inspector/v3/http_inspector.proto

```diff
@@ -18,4 +21,11 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 message HttpInspector {
   option (udpa.annotations.versioning).previous_message_type =
       "envoy.config.filter.listener.http_inspector.v2.HttpInspector";
+
+  // The number of bytes of the request the filter will initially buffer and
+  // inspect to determine the http version. If the http version cannot be
+  // identified with the current buffer, the filter will iteratively double
+  // the size up to 64KiB. If not specified, the initial buffer is 8KiB.
+  google.protobuf.UInt32Value initial_read_buffer_size = 1
+      [(validate.rules).uint32 = {lt: 65537 gt: 15}];
```

Comment:
Basically, I think the `initial_read_buffer_size` makes limited sense. Because we make the buffer size could increase dynamically now anyway.

Then the actaully limit is the max buffer size and no one will actually configure this field. Actually, you can remove the field and set a static min_read_buffer_size (8k, 4k, 2k) directly and the inspector also works fine.